tools/i18n/update-for-legacy-translations

```diff
@@ -0,0 +1,242 @@
+    "Who can post to the channel?": "Who can post to the stream?",
+    "Who can unsubscribe others from this channel?": "Who can unsubscribe others from this stream?",
+    "You are not currently subscribed to this channel.": "You are not currently subscribed to this stream.",
+    "You are not subscribed to any channels.": "You are not subscribed to any streams.",
+    "You are not subscribed to channel <z-stream-name></z-stream-name>.": "You are not subscribed to stream <z-stream-name></z-stream-name>.",
+    # "You aren't subscribed to this channel and nobody has talked about that yet!": "You aren't subscribed to this stream and nobody has talked about that yet!",
+    "You can use email to send messages to Zulip channels.": "You can use email to send messages to Zulip streams.",
+    "You cannot create a channel with no subscribers.": "You cannot create a stream with no subscribers.",
+    "You do not have permission to add other users to channels in this organization.": "You do not have permission to add other users to streams in this organization.",
+    "You do not have permission to move messages to another channel in this organization.": "You do not have permission to move messages to another stream in this organization.",
+    "You do not have permission to post in this channel.": "You do not have permission to post in this stream.",
+    # "You do not have permission to use <b>@{wildcard_mention_string}</b> mentions in this channel.": "You do not have permission to use <b>@{stream_wildcard_mention}</b> mentions in this stream.",
+    "You must be an organization administrator to create a channel without subscribing.": "You must be an organization administrator to create a stream without subscribing.",
+    "You subscribed to channel <z-stream-name></z-stream-name>.": "You subscribed to stream <z-stream-name></z-stream-name>.",
+    "You unsubscribed from channel <z-stream-name></z-stream-name>.": "You unsubscribed from stream <z-stream-name></z-stream-name>.",
+    "You're not subscribed to this channel. You will not be notified if other users reply to your message.": "You're not subscribed to this stream. You will not be notified if other users reply to your message.",
+    "Your message was sent to a channel you have muted.": "Your message was sent to a stream you have muted.",
+    "back to channels": "back to streams",
+}
+
+
+def get_json_filename(locale: str) -> str:
+    return f"locale/{locale}/translations.json"
+
+
+def get_legacy_filename(locale: str) -> str:
+    return f"locale/{locale}/legacy_stream_translations.json"
+
+
+def get_locales() -> List[str]:
+    output = check_output(["git", "ls-files", "locale"], text=True)
+    tracked_files = output.split()
+    regex = re.compile(r"locale/(\w+)/LC_MESSAGES/django.po")
+    locales = []
+    for tracked_file in tracked_files:
+        matched = regex.search(tracked_file)
+        if matched and matched.group(1) != "en_GB":
```

Comment:
The `get_locales` function is the same as in the other i18n scripts, except for this part. In this case, we won't want to have these legacy strings for any variant of English.